docs/docs/deploying-to-gitlab-pages.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Deploying to GitLab Pages
+---
+
+GitLab Pages are similar to GitHub Pages, perhaps even easier to setup. It also
+supports custom domain names and SSL certificates. The process of setting GitLab
+pages up is made a lot easier with GitLab's included continuous integration
+platform.
+
+Create a new GitLab repository, initialize your Gatsby project folder if you
+haven't already, and add the GitLab remote.
+
+```bash
+git init
+git remote add origin git@gitlab.com:examplerepository
+git add .
+git push -u origin master
+```
+
+You can deploy sites on GitLab Pages with or without a custom domain. If you choose to use the default setup (without a custom domain), or if you create a project site, you will need to setup your site with path prefixing. If adding a custom domain, you can skip the Path Prefix step, and remove `--prefix-paths` from the gitlab-ci.yml file.
+
+### Path Prefix
```

Comment:
Making this header more similar to the next one will help it be clearer. What about ## "Add Path Prefix to Gatsby"? Also, two hashes is enough for these headers.